app/worker.php

```diff
@@ -215,7 +215,7 @@
 
 $pools = $register->get('pools');
 $platform = new Appwrite();
-$args = $_SERVER['argv'];
+$args = $platform->getEnv('argv');
```

Comment:
Does Platform have `getEnv()`? I couldn't find it in https://github.com/utopia-php/platform/blob/main/src/Platform/Platform.php

Reply:
Now it will (:

Reply:
lets update composer with the new library version

Reply:
Lets update composer with https://github.com/utopia-php/platform/releases/tag/0.7.0